assets/python/ml/pyml:setup_variables_packages.pyi

```diff
@@ -158,3 +158,26 @@ class Context(object):
 context = Context()
 
 is_using_train_test_split = "is_using_train_test_split" in context and (context.load("is_using_train_test_split"))
+
+# Create a Class for a DummyScaler()
+class DummyScaler():
```

Comment:
Minor, but you don't need the opening parenthesis here. `class DummyScaler:` works fine